README.md

```diff
@@ -98,10 +117,10 @@ tunnel.listen(3128, function() {
 Parses Proxy-Authorization header and sets req.auth.username and req.auth.password properties.
 
 Further middleware should be added to accept or reject connections based on this authentication information.
-**Kind**: method of <code>[Tunnel](#tunnel)</code>	
+**Kind**: method of <code>[Tunnel](#tunnel)</code>
 **Summary**: Parse Proxy-Authorization header.
-**Access:** public  
-**Example**  
```

Comment:
You're gonna need to beat your vim into shape, as removing these trailing spaces messes up the markdown rendering